docs/arcc.md

```diff
@@ -0,0 +1,104 @@
+# nf-core/configs: ARCC Configuration
+
+The [Advanced Research Computing Center (ARCC)](http://www.uwyo.edu/arcc/) for the University
+of Wyoming (UW) has been set up to allow its users to utilize Nextflow with Singularity.
+
+## Getting Started
+
+First, you will need an account on ARCC, if you already have an account, skip ahead, otherwise
+please continue. To get an account, you will need to be a Principal Investigator (PI) or student
+at UW, or be sponsored by a UW PI. To learn more please visit [ARCC - HPC Account Requests](https://arccwiki.atlassian.net/wiki/spaces/DOCUMENTAT/pages/1913684148/Accounts+Access+and+Security).
+
+With an account in hand, you are ready to proceed.
+
+## Running Nextflow
+
+When using Nextflow on ARCC it is recommended you launch Nextflow on one of the compute nodes,
+instead of the login nodes. To do this you will use the `salloc` command to launch an
+[Interactive Job](https://arccwiki.atlassian.net/wiki/spaces/DOCUMENTAT/pages/1599078403/Start+Processing#Interactive-Jobs).
+
+Alongside `salloc`, please consider making use of [screen or tmux](https://arccwiki.atlassian.net/wiki/spaces/DOCUMENTAT/pages/1617494076/Screen+and+Tmux+Commands)
```

Comment:
I would maybe start with this before mentioning salloc in case someone is following the guide to the letter 😅